### ISSUE TITLE
Bug: Edit button isn't accessible

### ISSUE THREAD

**Issue by charleshalldesign** [GSA/digitalgov.gov#2146]
**Describe the bug**
The “Edit Tools” link (floating action button pattern) contains no accessible name and also cannot be reached by keyboard input due to positioning method 

**Steps to Reproduce**
1. Go to https://digital.gov/communities/accessibility/
2. Inspect Edit link in bottom right corner
     ```<a class="edit edit-open" href="#"><i class="far fa-edit" aria-hidden="true"></i></a>```
3. Observe no text node
4. Navigate entire page by keyboard
5. Observe the link cannot be focussed

**Screenshots**
(attached)
<img width="90" alt="Screen Shot 2020-03-04 at 1 07 11 PM" src="https://user-images.githubusercontent.com/3154739/75910105-4439fb80-5e1b-11ea-80c2-289c93a5e70a.png">

**Comment by charleshalldesign** [comment]
Commenting to address github-actions bot.
The issue persists.
Additional info: “Accessibility node not exposed. aria-hidden applied to ancestor.”
